main/src/ca/uwaterloo/flix/api/lsp/provider/completion/CompletionUtils.scala

```diff
@@ -52,6 +52,18 @@ object CompletionUtils {
 
   private def isUnitFunction(fparams: List[TypedAst.FormalParam]): Boolean = fparams.length == 1 && isUnitType(fparams(0).tpe)
 
+  def getLabelForEnumTags(name: String, cas: TypedAst.Case)(implicit flix: Flix): String = {
+    val regExpWithoutParen = """^[^()]*$""".r
+    val regExpWithParen = """^\(.*\)$""".r
+    val args = FormatType.formatType(cas.tpe)
+    args match {
+      case "Unit" => name
+      case regExpWithoutParen() => s"$name($args)"
+      case regExpWithParen() => s"$name$args"
+      case _ => name
+    }
+  }
```

Comment:
This is definitely not the right approach, but I was not able to come up with something better atm..

Reply:
However, it works perfectly.

Reply:
This is crazy indeed. Can you not just use the FormatType class?

Reply:
Hmm, I'll try that :)

Reply:
Currently not able to find anything from Format that can help.. 

We want `Enum.Tag(Type)`.
If we use `FormatType.formatType(tagTyp)` it would produce:
```
   case Unit => "Unit"
   case (a single type) => "type"
   case Tuple(x,y) => "(x, y)"
```
The issue is, when the type is `Unit`, we do not want to add any description. If it's just a single type, we need to add the parens. When it's a `Tuple` we do not want to add any parens..

Reply:
Maybe you can do a combination? Check I'd the type constructor is unit and otherwise use format type.